inference_schema/parameter_types/standard_py_parameter_type.py

```diff
@@ -57,6 +57,9 @@ def deserialize_input(self, input_data):
             input_data = parser.parse(input_data).timetz()
         elif self.sample_data_type is bytearray or (sys.version_info[0] == 3 and self.sample_data_type is bytes):
             input_data = base64.b64decode(input_data.encode('utf-8'))
+        elif self.sample_data_type is float and type(input_data) is int:
+            # Allow users to pass ints when expecting floats, for convenience
+            pass
         if not isinstance(input_data, self.sample_data_type):
```

Comment:
Won't this check still fail? Seems like we should explicitly convert the int to a float in this case.

Reply:
Oh, I meant to put this as an initial if check before the "isinstance" check on 63. And this is why I'm adding a test....

Let me make that change as I originally intended and I'll ping you again for an opinion. I'm not against casting it to a float, but will go based on what I originally intended